src/treeitem_dependency.mli

```diff
@@ -0,0 +1,26 @@
+type t =
+  | Package : Sandbox.Package.t * t option -> t
+  | Sandbox : Sandbox.t -> t
```

Comment:
It's weird that this type is reused between the sandbox and switches treeview. Is it correct that that in the sandbox view, only `Package` can appear? If that's the case, we should really be dealing with two different types for the different views.

Reply:
I split the implementations (actually re-implemented it with the new VSCode API), so the types are not shared anymore.